crates/cairo-annotations/src/trace_data.rs

```diff
@@ -43,6 +43,8 @@ pub struct CairoExecutionInfo {
     /// Path to a file with serialized `ContractClass` or `VersionedProgram`.
     pub source_sierra_path: Utf8PathBuf,
     pub casm_level_info: CasmLevelInfo,
+    /// `enable-gas` option from [cairo] section in `Scarb.toml`
+    pub enable_gas: Option<bool>,
```

Comment:
Why not just bool? 🤔

Reply:
Do we always know how to treat option::none? 🤔

Reply:
It depends. In this special case, if it's not present in the trace file I just set it to `true` in profiler (as it's the only one that uses this struct, apart from snforge and now scarb). Otherwise I would get serialization errors, if someone tried to use older trace file.

Reply:
Oh, so it's for serde compatbility - makes sense then!